### ISSUE TITLE
Rename Wall-E to Wall·E

### ISSUE THREAD

**Issue by PaymonJ** [CSSS/wall_e#71]
[A la the movie poster[(https://m.media-amazon.com/images/M/MV5BMjExMTg5OTU0NF5BMl5BanBnXkFtZTcwMjMxMzMzMw@@._V1_SY1000_CR0,0,674,1000_AL_.jpg).

**Comment by modernNeo** [comment]
I disagree only cause its not on the keyboard and had to be copy pasted
